Marlin/src/lcd/ultralcd.cpp

```diff
@@ -1553,20 +1553,23 @@ void MarlinUI::update() {
     //  - On edit screens, touch Up Half for -,  Bottom Half to +
     //
     void MarlinUI::screen_click(const uint8_t row, const uint8_t col, const uint8_t, const uint8_t) {
+      const millis_t now = millis();
+      if (PENDING(now, next_button_update_ms)) return;
+      next_button_update_ms = now + repeat_delay;    // Assume the repeat delay
       const int8_t xdir = col < (LCD_WIDTH ) / 2 ? -1 : 1,
                    ydir = row < (LCD_HEIGHT) / 2 ? -1 : 1;
       if (on_edit_screen)
-        encoderDiff = epps * ydir;
+        encoderDiff = epps * ydir * 2;
```

Comment:
Maybe this 2 should be `ENCODER_PULSES_PER_STEP`…?

Reply:
just : `encoderDiff = ydir;` ... But I didn't test it.